### PR TITLE
Add Android for Web developers page

### PR DESCRIPTION
**Pull request by joycetoh8**
Changes proposed in this pull request:

- Add a new page with key Android concepts web developers should know if they're publishing PWAs to Play
- Rearrange the Digital Asset Links related content in TWA Quickstart Guide and pull some out to the new page
- Remove outdated info about Bubblewrap generating `assetlinks.json` file

### CODE REVIEW

**Review by andreban — COMMENTED** [review]
Dropped some notes

File: site/en/docs/android/trusted-web-activity/android-for-web-devs/index.md
Comment (joycetoh8) [reply site/en/docs/android/trusted-web-activity/android-for-web-devs/index.md:7]: Oops forgot about this. Thanks for pointing it out!

File: site/en/docs/android/trusted-web-activity/android-for-web-devs/index.md
Comment (joycetoh8) [reply site/en/docs/android/trusted-web-activity/android-for-web-devs/index.md]: done!

File: site/en/docs/android/trusted-web-activity/android-for-web-devs/index.md
Comment (joycetoh8) [reply site/en/docs/android/trusted-web-activity/android-for-web-devs/index.md]: I moved APK to after the note about AABs being required soon.

File: site/en/docs/android/trusted-web-activity/android-for-web-devs/index.md
Comment (joycetoh8) [reply site/en/docs/android/trusted-web-activity/android-for-web-devs/index.md]: Yep, this is good to clarify. Added a sentence about this.

File: site/en/docs/android/trusted-web-activity/quick-start/index.md
Comment (PEConn) [inline site/en/docs/android/trusted-web-activity/quick-start/index.md:127]: nit: I think there should be a newline before headers in markdown.

**Review by PEConn — APPROVED** [review]

File: site/en/docs/android/trusted-web-activity/quick-start/index.md
Comment (joycetoh8) [reply site/en/docs/android/trusted-web-activity/quick-start/index.md:128]: Thanks, Peter. Good catch!

**Comment by andreban** [comment]
Thanks, Joyce. Content looks good - Can you check the changes changes requested by the review bot at https://github.com/GoogleChrome/developer.chrome.com/pull/1043#issuecomment-887966287?

File: site/en/docs/android/trusted-web-activity/quick-start/index.md
Comment (samthor) [inline site/en/docs/android/trusted-web-activity/quick-start/index.md:130]: I approved another change which removed this section, so you'll probably need to rebase.

**Review by samthor — APPROVED** [review]

**Review by andreban — APPROVED** [review]
LGTM, Thanks!

File: site/en/docs/android/trusted-web-activity/quick-start/index.md
Comment (joycetoh8) [reply site/en/docs/android/trusted-web-activity/quick-start/index.md]: Thanks for the heads up, Sam! I think I've resolved any conflicts with main now.

**Comment by joycetoh8** [comment]
@samthor How do I get a maintainer to approve the workflows? It'd be great if we could get this content merged soon, thanks!

**Comment by samthor** [comment]
@joycetoh8 I've set them to run. I'll submit after it's done.

I think you should be added to more Google groups on GitHub, so this should just work for you...

File: site/en/docs/android/trusted-web-activity/quick-start/index.md
Comment (samthor) [inline site/en/docs/android/trusted-web-activity/quick-start/index.md:43]: I'm happy you're fixing this link but this feels like a failure: there should be a page at /docs/android/custom-tabs instead, but it 404's right now.

**Pull request merged**
